nixos/modules/services/networking/mosh.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg  = config.services.mosh;
```

Comment:
I think `programs` would be more appropriate rather than `services` because no real service will be defined in this module.